macosx/kext/AudioEngine.cpp

```diff
@@ -125,7 +126,8 @@ bool kXAudioEngine::initHardware(IOService *provider)
     if(hw->is_10k2)
     {
 			// setSampleOffset(28); // 28 samples
-		setSampleLatency(28+1);
+		//setSampleLatency(28+1);
+        setSampleOffset(40);
```

Comment:
28 was taken from 10k1/10k2 datasheet specs. Where '40' comes from?

Reply:
With such a value the recorded sound is normal for me.